contrib/initramfs/scripts/zfs

```diff
@@ -190,11 +190,11 @@ get_pools()
 # Import given pool $1
 import_pool()
 {
-	pool="$1"
+	import_pool="$1"
```

Comment:
Technically not part of the original problem, but if we're going to rename parameter variables, better do it everywhere just to be safe and be consistent.